src/main/scala/tilelink/Xbar.scala

```diff
@@ -30,8 +31,11 @@ object ForceFanout
 private case class ForceFanoutParams(a: Boolean, b: Boolean, c: Boolean, d: Boolean, e: Boolean)
 private case object ForceFanoutKey extends Field(ForceFanoutParams(false, false, false, false, false))
 
-class TLXbar(policy: TLArbiter.Policy = TLArbiter.roundRobin)(implicit p: Parameters) extends LazyModule
+class TLXbar(
+  policy: TLArbiter.Policy = TLArbiter.roundRobin // use option "roundRobin" which has similar property of MUX
+  )(implicit p: Parameters) extends LazyModule
```

Comment:
What does this comment mean? Also why does arbiter choice affect decoder

Reply:
yes, these are some code from another branch, I just removed them, thanks